jobs/JGLOBAL_WAVE_POST_PNT

```diff
@@ -9,9 +9,15 @@ export errchk=${errchk:-err_chk}
 export MP_PULSE=0
 
 # Set COM Paths and GETGES environment
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_WAVE_PREP COM_WAVE_HISTORY COM_WAVE_STATION
-
-if [[ ! -d ${COM_WAVE_STATION} ]]; then mkdir -p "${COM_WAVE_STATION}"; fi
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+   COMIN_WAVE_PREP:COM_WAVE_PREP_TMPL \
+   COMIN_WAVE_HISTORY:COM_WAVE_HISTORY_TMPL \
+   COMOUT_WAVE_HISTORY:COM_WAVE_HISTORY_TMPL \
```

Comment:
Again, no need for COMOUT_WAVE_HISTORY, unless I'm missing something.

Reply:
This was also resolved.